packages/cli/src/tools/packageManager.js

```diff
@@ -11,6 +11,35 @@ type Options = {|
 
 let projectDir;
 
+const packageManagers = {
+  yarn: {
+    add: ['add'],
+    addDev: ['add', '-D'],
+    remove: ['remove'],
+    install: ['install'],
+  },
+  npm: {
+    add: ['install', '--save', '--save-exact'],
+    addDev: ['install', '--save-dev', '--save-exact'],
+    remove: ['uninstall', '--save'],
+    install: ['install'],
+  },
+};
+
+function configurePackageManager(
+  packageNames: Array<string>,
+  options?: Options,
+  action: string,
```

Comment:
please make it an enum as I already noted: https://github.com/react-native-community/cli/pull/433#discussion_r296023034